pkg/test/framework/components/istio/istio.go

```diff
@@ -75,6 +75,10 @@ func Setup(i *Instance, cfn SetupConfigFn, ctxFns ...SetupContextFn) resource.Se
 			return err
 		}
 		if cfn != nil {
+			if !cfg.DeployIstio {
```

Comment:
I do not think we should do this, it will break development workflows where we install 1x then rerun tests many times.

Reply:
Also, it seems the vast majority of tests have *some* function here, even if its not doing much. This is why in the past we explicitly listed them as CustomSetup